docs/spec/README.md

```diff
@@ -1,12 +1,12 @@
 # Helm Controller
 
-The Helm Controller is a Kubernetes operator, allowing one to declaratively manage Helm chart
+The Helm Controller is a Kubernetes controller, allowing one to declaratively manage Helm chart
```

Comment:
```suggestion
The Helm Controller is a [Kubernetes operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/), allowing one to declaratively manage Helm chart
```

The helm-controller is a controller but also an operator (all operators are controllers but not all controllers are operators), what makes the helm-controller an operator is the fact that it has a single-app focus (Helm).

See: https://kubernetes.io/docs/concepts/extend-kubernetes/operator/

Reply:
done